packages/core/src/video/VideoForDevelopment.tsx

```diff
@@ -69,7 +74,22 @@ const VideoForDevelopmentRefForwardingFunction: React.ForwardRefRenderFunction<
 		acceptableTimeshift: acceptableTimeshift ?? DEFAULT_ACCEPTABLE_TIMESHIFT,
 	});
 
-	const actualSrc = usePreload(src as string);
+	const duration = parentSequence
+		? Math.min(parentSequence.durationInFrames, durationInFrames)
+		: durationInFrames;
+
+	let actualSrc = usePreload(src as string);
+
+	const shouldAppendFragment = !actualSrc.startsWith('data:') && !actualSrc.includes('#t=')
+	if (shouldAppendFragment) {
+		if (typeof shouldBeTime === 'number' && Number.isFinite(shouldBeTime)) {
```

Comment:
Hmm, now it looks like:

- instead of the start point of the video, it takes the current time
- instead of the end point of the video, it takes the duration of the sequence

I recommend taking a look at how the video layer gets displayed in the timeline: https://github.com/remotion-dev/remotion/blob/main/packages/core/src/use-media-in-timeline.ts

You can also start the preview in the `example` folder and open the DevTools to verify that the right fragment is calculated.